src/reducers/index.js

```diff
@@ -1,6 +1,8 @@
 import { combineReducers } from 'redux';
 import { charsReducer } from './starWarsReducer';
+import { fetchingReducer } from './fetchingReducer'
 
 export default combineReducers({
-  charsReducer
+  charsReducer,
```

Comment:
I think the Readme got you confused here - you split it into 2 reducers where one was enough.